src/Caching/Abstractions/ref/Microsoft.Extensions.Caching.Abstractions.csproj

```diff
@@ -6,9 +6,11 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <Compile Include="Microsoft.Extensions.Caching.Abstractions.netstandard2.0.cs" />
     <Reference Include="Microsoft.Extensions.Primitives"  />
+    <Reference Include="Microsoft.CodeAnalysis.FxCopAnalyzers"  />
```

Comment:
Should this be showing up in the ref assemblies? Will it show up as a dependency from the official package?

Reply:
I do not think it should.